Frameworks/include/CoreGraphics/CGFontInternal.h

```diff
@@ -0,0 +1,20 @@
+//******************************************************************************
```

Comment:
nit: move this header to just be in Frameworks/CoreGraphics - it's not needed cross-module